react/features/settings/functions.js

```diff
@@ -112,7 +115,9 @@ export function getMoreTabProps(stateful: Object | Function) {
             && isLocalParticipantModerator(state));
 
     return {
+        currentFramerate: `${framerate} frames-per-second`,
```

Comment:
Can we do i18n for `frames-per-second` or at least change it to fps maybe ...?

Reply:
I can translation for frames-per-second. This was in figma that I got from Tibby, so I believe product wanted to keep it that way.